### PR TITLE
Added coordinates to pin on map

### PR DESCRIPTION
**Pull request by ganginenisarath**
@githubteacher  Added **lon.** and *lat.*

closess #3017

### CODE REVIEW

**Comment by angelotrivelli** [comment]
Looks good to me.

**Comment by Apoorva-A** [comment]
LGTM! Awesome!!

**Comment by crichID** [comment]
@pierreneter thank you for trying to help keep this repository tidy, but we ask that you please **do not** close or merge pull requests that you did not open. Please leave these activities so each learner may complete their own tasks!

File: sarath.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ ---
```
Comment (crichID) [inline sarath.json]: @ganginenisarath the reason the tests are failing is that you have a space at the beginning of each line. If you click the Files changed tab, you will see a small pencil at the top right of the file. Click that and you will be back in edit mode! Please let me know if I can help!

**Comment by crichID** [comment]
@ganginenisarath you will also need to place your file in the `_pins` directory. To do this, access the online editor and then place your cursor at the beginning of the file name. Type `_pins/` before your filename and this place it in the correct directory.

**Comment by crichID** [comment]
@ganginenisarath you have a great start in this file but there isn't enough information in the coordinates to me to finish it! I am going to go ahead and close this pull request from now. If you would like to finish this activity, please feel free to re-open this pull request or start a new one!

**Pull request closed without merging**